Xamarin.Forms.Platform.iOS/ContextActionCell.cs

```diff
@@ -59,6 +60,13 @@ public bool IsOpen
 			get { return ScrollDelegate.IsOpen; }
 		}
 
+		public bool IsTouchInDisabledButton(PointF pointf)
+		{
+			double totalButtonWidth = _buttons.Aggregate(0.0, (current, uiButton) => current + uiButton.Frame.Width);
```

Comment:
I haven't tested this, but I wonder if it would make sense to cache this value instead of calculating it every time...?